skiko/src/awtMain/kotlin/org/jetbrains/skiko/redrawer/MetalRedrawer.kt

```diff
@@ -63,8 +84,8 @@ internal class MetalRedrawer(
     override fun dispose() = synchronized(drawLock) {
         frameDispatcher.cancel()
         contextHandler.dispose()
-        disposeDevice(device)
-        device = 0
+        disposeDevice(device.ptr)
+        device = MetalDevice(0)
```

Comment:
Is it something like NULL pointer ?

Reply:
I think so, don't know why null wasn't used

Reply:
Maybe `null` makes more sense, indeed.

Reply:
we can create const val NULL: Long = 0 for this cases

Reply:
Or NULL_PTR: Long = 0

Reply:
Changed it to null, please check the latest commit